.gitignore

```diff
@@ -11,7 +11,6 @@ __pycache__
 *.pytest_cache
 build
 htmlcov
-
```

Comment:
Please remove this change (delete blank line) so that this file is not part of the PR at all.

Reply:
sure

Reply:
did it.  `.gitignore` is not the PR now.